python/_tskitmodule.c

```diff
@@ -6430,7 +6466,9 @@ static PyMethodDef TableCollection_methods[] = {
         METH_VARARGS|METH_KEYWORDS,
         "Returns an edge table linking samples to a set of specified ancestors." },
     {"subset", (PyCFunction) TableCollection_subset, METH_VARARGS,
-        "Subsets the tree sequence to a set of nodes." },
+        "Subsets the table collection to a set of nodes." },
+    {"union", (PyCFunction) TableCollection_union, METH_VARARGS,
+        "Forms the union of two table collections." },
```

Comment:
How about "Adds to this table collection the portions of another table collection that are not shared with this one."